Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:16-alpine
+# Installing libvips-dev for sharp Compatibility
+RUN apk update && apk add --no-cache build-base gcc autoconf automake zlib-dev libpng-dev nasm bash vips-dev
+ARG NODE_ENV=development
+ENV NODE_ENV=${NODE_ENV}
+
+WORKDIR /opt/
+COPY package.json yarn.lock ./
+RUN yarn config set network-timeout 600000 -g && yarn install
```

Comment:
why the `yarn config set network-timeout`?

Reply:
It reduces the chances of CI failing due to yarn issues. Per docs of strapi, it is suggested to add this as some dependencies can be large. https://docs.strapi.io/dev-docs/installation/docker

Reply:
So, the timeout is 10 min?

Reply:
Yes, that is correct